README.md

```diff
@@ -132,4 +157,5 @@ Finally, link to your repo on github in the space below.
 
 Your Answer:
 ```text
+  Project 1: https://github.com/codedre/project-1
```

Comment:
Impressive.

I recommend that you break your bronze stories down even smaller.  Separate game play and winning requirement stories.